wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/EnumType.kt

```diff
@@ -29,6 +29,7 @@ data class EnumType(
   override val documentation: String,
   override val name: String,
   val constants: List<EnumConstant>,
+  val defaultConstantValue: Int,
```

Comment:
What is a default constant value?

Reply:
this is also breaking APIs

Reply:
It's the `tag` value of an enum. The "default" value for a proto is defined to always be the first case of that enum.
Since constants can (and are) pruned, I am persisting this at initialization time.

In proto3 land, it should always be `0` as the spec requires the first constant to have a value of `0`.
In proto2 land, there are no guarantees.

Reply:
Sorry, let me add to the PR description